pkg/resourcemanager/mysql/mysqluser/mysqluser.go

```diff
@@ -126,30 +125,26 @@ func (s *MySqlUserManager) CreateUser(ctx context.Context, server string, secret
 }
 
 // UserExists checks if db contains user
-func (s *MySqlUserManager) UserExists(ctx context.Context, db *sql.DB, username string, server string) (bool, error) {
-
-	//tsql := fmt.Sprintf("SELECT * FROM mysql.user WHERE (Host = '%s') and (User = '%s')", server, username)
+func (s *MySqlUserManager) UserExists(ctx context.Context, db *sql.DB, username string) (bool, error) {
 
-	err := db.QueryRowContext(ctx, "SELECT * FROM mysql.user WHERE (Host = $1) and (User = $2)", server, username)
+	err := db.QueryRowContext(ctx, "SELECT * FROM mysql.user WHERE User = $1", username)
 	//err := db.ExecContext(ctx, tsql)
 
 	if err != nil {
 		return false, nil
 	}
 	return true, nil
 
-	//rows, err := res.RowsAffected()
-	//return rows > 0, err
-
 }
 
 // DropUser drops a user from db
-func (s *MySqlUserManager) DropUser(ctx context.Context, db *sql.DB, user string, server string) error {
-	tsql := fmt.Sprintf("DROP USER IF EXISTS '%s'@'%s'", user, server)
+func (s *MySqlUserManager) DropUser(ctx context.Context, db *sql.DB, user string) error {
+
 	if err := helpers.FindBadChars(user); err != nil {
 		return fmt.Errorf("Problem found with username: %v", err)
 	}
-	_, err := db.ExecContext(ctx, tsql)
+	tsql := "DROP USER IF EXISTS ?"
+	_, err := db.ExecContext(ctx, tsql, user)
```

Comment:
Minor: the tsql string seems kind of redundant now...
```suggestion
	_, err := db.ExecContext(ctx, "DROP USER IF EXISTS ?", user)
```